### PR TITLE
Update venetian.xml to 7.8.8

### PR DESCRIPTION
**Pull request by conky77**

### CODE REVIEW

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 1]
```diff
@@ -950,6 +949,7 @@ Te poi definire vari deƚimitatori de coƚona uxando spasi par spartire i divers
                     <Item id="6331" name="Visuaƚixa soƚo nòme file ne ƚa bara de&apos;l tìtoƚo"/>
                     <Item id="6334" name="Riƚeva otomàticamente codefega caràtari"/>					
                     <Item id="6314" name="Uxa caràtare a spasiatura fisa ne ƚa fenèstra de riserca (Te devi inviare de novo Notepad++)" />
+					<Item id="6349" name="Uxa DirectWrite (Poderìa mejorare el rendering de i carateri speciali, cogna inviare de novo Notepad++)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:952]: You may want to align this line.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 2]
```diff
@@ -1268,11 +1273,15 @@ Ruma en tuti i file a ecesion exe, obj &amp;&amp; log:
             <replace-in-files-confirm-title value="Sito sicuro?"/>
             <replace-in-files-confirm-directory value="Sito sicuro de voler sostituire tute cuante le ocorense in:"/>
             <replace-in-files-confirm-filetype value="Par i tipi de file:"/>
-            <find-result-caption value="Cata rixultadi"/>
+            <replace-in-open-docs-confirm-title value="Sito sicuro?"/>
+            <replace-in-open-docs-confirm-message value="Sito sicuro de voler sostituire tute cuante le ocorense en tuti cuanti i documenti verti?"/>            
+			<find-result-caption value="Cata rixultadi"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1278]: You may want to align this line.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 3]
```diff
@@ -1268,11 +1273,15 @@ Ruma en tuti i file a ecesion exe, obj &amp;&amp; log:
...
             <find-result-title value="Cata"/>
             <find-result-title-info value="($INT_REPLACE1$ ocorense in $INT_REPLACE2$ file de $INT_REPLACE3$ sercà)"/>
+			<find-result-title-info-selections value="($INT_REPLACE1$ ocorense in $INT_REPLACE2$ selesioni de $INT_REPLACE3$ sercà)"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1281]: You may want to align this line.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 4]
```diff
@@ -1268,11 +1273,15 @@ Ruma en tuti i file a ecesion exe, obj &amp;&amp; log:
...
             <find-result-title-info-extra value=" - Modaƚità de filtro ƚinea: mostra soƚo i rixultati filtrà"/>
             <find-result-hits value="($INT_REPLACE$ ocorense)"/>
+			<find-regex-zero-length-match value="corispondensa de longhesa zero" />
```
Comment (donho) [inline PowerEditor/installer/nativeLang/venetian.xml:1284]: You may want to align this line.

**Comment by conky77** [comment]
done

**Pull request closed without merging**